src/selectors/per-thread/markers.js

```diff
@@ -360,25 +360,50 @@ export function getMarkerSelectorsPerThread(
   );
 
   /**
-   * This filters out network markers from the list of all markers, so that
-   * they'll be displayed in the marker chart.
+   * The marker chart ignores the preview selection, as it handles its own zooming
+   * behavior. It uses the SearchFilteredMarkerIndexes instead. It shows markers
+   * that use the "marker-chart" schema location, plus it shows markers that have
+   * no schema, in order to be as permissive as possible.
    */
   const getMarkerChartMarkerIndexes: Selector<MarkerIndex[]> = createSelector(
     getMarkerGetter,
-    getCommittedRangeAndTabFilteredMarkerIndexes,
-    MarkerData.filterForMarkerChart
```

Comment:
nit: you can remove this function as it was used only here.

Reply:
Ah, we should add `import/no-unused-modules`